test/test_onnx.py

```diff
@@ -65,9 +65,9 @@ def to_numpy(tensor):
                 torch.testing.assert_allclose(outputs[i], ort_outs[i], rtol=1e-03, atol=1e-05)
             except AssertionError as error:
                 if tolerate_small_mismatch:
-                    assert ("(0.00%)" in str(error)), str(error)
+                    self.assertIn("(0.00%)", str(error), str(error))
                 else:
-                    assert False, str(error)
+                    self.assertTrue(False, str(error))
```

Comment:
What about making this a `raise AssertionError(str(error))` instead of the fake assert.

Reply:
Definitely cleaner than assert on boolean `False`. 
Initially, I thought you guys were keeping it for compatibility purposes, but I will update the PR accordingly!

Since the `error` in context is already an `AssertionError`, we can even go for a bare `raise` to avoid the string casting and error type redundancy.